### DISCUSSION TITLE
How to disable lazyvim's which-key keymap?

### DISCUSSION THREAD

**Discussion by lucasrabiec** [LazyVim/LazyVim#373] [Q&A] [answered]
I removed all `+tabs` keymaps and now I want to remove it also from the which-key panel.

I have `which-key.lua` with following code:
```lua
return {
  "folke/which-key.nvim",
  config = function()
    require("which-key").register({
      ["<leader><tab>"] = "which_key_ignore",
    })
  end,
}
```
It removes `+tab` from the panel but it also resets other key names to `+prefix`. I am guessing it is because calling register overwrites old keymappings. So how I can remove only one keybinding from which_key panel?

**Comment by folke** [comment, marked as answer]
you're overriding the whole `config` block this way, which is probably not what you want.

Just do what you do somewhere else in the code. Maybe in your `lua/config/keymaps.lua` file

**Reply by lucasrabiec** [reply]
Yep, works. TY
